language-design.md

```diff
@@ -70,7 +158,17 @@ what ways do the themes apply to the study and creation of DSLs?
 
 **Response**
 
+The themes would not apply to DSLs as much. Because DSLs are domain
+specific, it’s likely that most DSLs are the only language to serving the domain. 
+Thus, there is already less or no competition between languages for DSLs. 
```

Comment:
This is true, but I think it's possible for there to be bias against DSLs in general, if not against a specific one. For example, I think there's a tendency for people to think of DSLs as "not real languages", especially those that aren't Turing-complete and really can't be used for anything but their domain. HTML may be an example of this, and I think it's possible that some of the prejudice against front-end devs stems from this idea.